include/topgg/client.hpp

```diff
@@ -24,7 +24,7 @@ class Client {
 
  private:
   std::string m_token;
-  std::string m_baseUrl = "https://top.gg/api/";
+  std::string m_baseUrl = "https://top.gg/api/";1233053525509013685
```

Comment:
not sure what this is supposed to do, at best it is a warning and at worst a compile error